server/utils/populate-all.js

```diff
@@ -59,12 +69,12 @@ function populateAll(schema, maxDepth = 10) {
  * @param {array} translatedFieldTypes The types of fields that are translated
```

Comment:
JSDoc does not correspond to implementation.
```suggestion
 * @param {object} options - options for recursion and population
 * @param {number} options.maxDepth - maximum recursive depth
 * @param {boolean} options.populateMedia - see {@link populateAll}
```
The line above this needs to be replaced/removed as well, not sure how to add it to this Suggestion